modules/migration-about-migration-hooks.adoc

```diff
@@ -17,16 +15,14 @@ A migration hook runs on a source or a target cluster at one of the following mi
 * `PreRestore`: Before resources are restored on the target cluster
 * `PostRestore`: After resources are restored on the target cluster
 
```

Comment:
I think we should add a one-line description of `PreBackup` and `PostBackup` hooks as well.